sinatra/app.rb

```diff
@@ -0,0 +1,49 @@
+# sinatra setup prototype
+# OSUOSL March 2016
+
+require 'rubygems'
+require 'bundler/setup'
+require 'sinatra'
+require 'json'
+require 'date'
+
+# preprocess the json from server
+# TODO: change data source to redis
+file = File.read('sample.json')
+data = JSON.parse(file)
+
+# default helloworld
+get '/' do
+  "Hello World!"
+end
+
+# returns bulk data
+get '/data' do
+  content_type:json
+  data.to_json
```

Comment:
I understand ruby allows us to omit the 'return' statement in favor of just returning the last thing in a block, but I'm in favor of *explicitly* returning data as it is more clear to the reader.

tldr, let's add return statements.